microsoft-365-ops/3.0/modules/ROOT/nav.adoc

```diff
@@ -0,0 +1,7 @@
+.xref:index.adoc[Microsoft Dynamics 365 For Operations Connector]
+* xref:index.adoc[About Microsoft Dynamics 365 For Finance and Operations Connector]
+* xref:microsoft-365-finance-operations-connector-upgrade.adoc[Upgrade Microsoft Dynamics 365 For Finance and Operations Connector to Version 3.0]
+* xref:microsoft-365-finance-operations-connector-studio.adoc[Use Studio to Configure the Connector]
```

Comment:
I usually spell out the connector name but I know it's long

Reply:
Yes, I usually do too but in this case... I just put it in the more important links.